src/atoms/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import { COLORS, FONT_SIZE } from "../constants/constants";
+
+export const Button = styled.button`
+  background: ${COLORS.MIDDLE_GRAY};
+  color: ${COLORS.LIGHT_GRAY};
+  font-size: ${FONT_SIZE.BASE};
+  padding: 10px;
+  border: 2px solid ${COLORS.LIGHT_GRAY};
+  border-radius: 3px;
+  cursor: pointer;
+
+  &:hover {
+    background: ${COLORS.GRAY};
+  }
+`;
```

Comment:
remove irrelevant files that should not be here like Input Link Card Drawer 
Only what corresponds to Button